interpreter/terminal_interface/magic_commands.py

```diff
@@ -172,6 +175,79 @@ def handle_count_tokens(self, prompt):
 
     display_markdown_message("\n".join(outputs))
 
+def get_desktop_path():
+    # For Windows
+    if os.name == 'nt':
+        desktop = os.path.join(os.path.join(os.environ['USERPROFILE']), 'Desktop')
+    # For MacOS and Linux
+    else:
+        desktop = os.path.join(os.path.join(os.path.expanduser('~')), 'Desktop')
+    return desktop
```

Comment:
Should we use the download folder instead of the Desktop?

Reply:
Yeah that makes a lot more sense than desktop. I can change that

Reply:
Changed it!